ldi-core/ldes-client-v2/starting-node/src/test/java/ldes/client/startingtreenode/StartingTreeNodeFinderIT.java

```diff
@@ -1,4 +1,4 @@
-package ldes.client.startingnode;
+package ldes.client.startingtreenode;
```

Comment:
based on the package naming convention it should be be.vlaanderen.informatievlaanderen.ldes.ldi.client.startingtreenode